app/views/layouts/find_layout.html.erb

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template">
+<head>
+  <title><%= content_for(:page_title) %> <%= t("service_name.find") %> - GOV.UK</title>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/favicon.ico" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-mask-icon.svg", rel: "mask-icon", type: "image/svg", color: "#0b0c0c" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon.png", rel: "apple-touch-icon", type: "image/png" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-152x152.png", rel: "apple-touch-icon", type: "image/png", size: "152x152" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-167x167.png", rel: "apple-touch-icon", type: "image/png", size: "167x167" %>
+  <%= favicon_link_tag "govuk-frontend/govuk/assets/images/govuk-apple-touch-icon-180x180.png", rel: "apple-touch-icon", type: "image/png", size: "180x180" %>
+  <%= stylesheet_link_tag "accessible-autocomplete/dist/accessible-autocomplete.min" %>
+  <%= stylesheet_link_tag "application" %>
+  <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+</head>
+
+<body class="govuk-template__body ">
+
+<%= render FindInterface::Header::View.new(
+  service_name: t("service_name.find"),
+) %>
+
+<div class="govuk-width-container">
+  <%= render FindInterface::PhaseBanner::View.new %>
+  <main class="govuk-main-wrapper" id="main-content" role="main"></main>
```

Comment:
Are we missing a <%= yield %> here?

Reply:
Yes

Reply:
Added https://github.com/DFE-Digital/publish-teacher-training/pull/2922/commits/a101c36b60232ea14e89901cad307580e26b345b

Reply:
Should the yield sit within the main ?

Reply:
Yes 😅 done https://github.com/DFE-Digital/publish-teacher-training/pull/2922/commits/562fbc914183fc385be103666b6e937f971f015b